openstack/blockstorage/extensions/volumeactions/results.go

```diff
@@ -50,6 +55,101 @@ func (r commonResult) Extract() (map[string]interface{}, error) {
 	return s.ConnectionInfo, err
 }
 
+// VolumeImage contains information about volume upload to an image service.
```

Comment:
This is what I wanted to address. What does this type represent (descriptively if possible)?